src/gui/components/ChannelBoot.tsx

```diff
@@ -266,7 +267,28 @@ const terminalFlow = ({
                         style={{ padding: '0 4px', cursor: 'pointer' }}
                         onClick={paste}
                     >
-                        content_paste_go
+                        content_paste_go{' '}
+                    </span>
+                    <span
+                        className={termstyles.link}
+                        onClick={() => {
+                            new Promise<void>((resolve, reject) => {
+                                hardReset()
+                                    .then(() => sleep(1000))
+                                    .then(() => {
+                                        window.location.reload();
+                                        resolve();
+                                    })
+                                    .catch((err) => {
+                                        alert(`hard-reset-fail: ${err}`);
+                                        reject(err);
+                                    });
+                            }).catch((err) => {
+                                console.error('Promise handling error:', err);
+                            });
+                        }}
+                    >
+                        (or go back)
```

Comment:
I feel like the parentheses and the icon and colon are all a bit muddled up.... and we should probably underline the "go back" so it's clear you can click it

so maybe it should be more like:

Paste a Playerchain key to join, or  [go back](#example) [paste-icon]:

Reply:
Fixed:
![image](https://github.com/user-attachments/assets/72c1ed4f-3181-401d-bdf2-d19a1eb0b301)
![image](https://github.com/user-attachments/assets/8f4502d4-a62f-4b38-92b4-0881e9d4e7fa)